mailbox/client_conn.go

```diff
@@ -156,7 +160,7 @@ func NewClientConn(ctx context.Context, sid [64]byte, serverHost string,
 	c := &ClientConn{
 		transport: transport,
 		gbnOptions: []gbn.Option{
-			gbn.WithTimeout(gbnTimeout),
```

Comment:
Yes this was ment to be a replace. If we use `WithTimeout`, the timeout will be set to a static value, and won't change dynamically even if we set a multiplier. So it's setting it as a static value, or letting it be dynamic :)

Reply:
gotcha! (btw - for comments like this where it's a question & not like a small nit or something - rather let the reviewer click "resolve comment" so that the reviewer doesnt miss the response to their question :)

Reply:
shall we then rather rename `WithTimeout` to `WithStaticTimeout` to make this clear?

Reply:
Ok true, I've updated this with the latest push. Let me know if it addresses your feedback :)

Reply:
> gotcha! (btw - for comments like this where it's a question & not like a small nit or something - rather let the reviewer click "resolve comment" so that the reviewer doesnt miss the response to their question :)

Ok good to know, thanks! Will keep that in mind from now on :)!